app/main.js

```diff
@@ -9,11 +11,13 @@ import App from './app.jsx';
 import Store from './reducers/index.js';
 import { createStore, } from 'redux';
 
+const socket = io();
+
 const store = createStore(Store, window.devToolsExtension && window.devToolsExtension());
 
 ReactDOM.render(
   <Provider store={ store }>
-    <App />
+    <App socket={ socket } />
```

Comment:
Okay, so I know nothing is happening yet, but I think setup / management of sockets will not live inside a component, I can and does for some react projects, but having state in components (like a socket) is kinda a redux anti-pattern. For now can we make another folder called socket.js in the client, although it will be a stub. You can require it here so that it runs and the connection gets opened, but do not pass it on to your app.

@emilyb7 I think we discussed sockets might play with redux (although I have not looked up if I am correct about this)

Also, happy to let this slide if what I am saying is annoying, I know that this code is unfinished (would be nice not to have it in a pull..)